packages/graphql-server/src/functions/graphql.ts

```diff
@@ -58,155 +35,11 @@ export const createGraphQLHandler = ({
   armorConfig,
   allowedOperations,
   allowIntrospection,
+  allowGraphiQL,
   defaultError = 'Something went wrong.',
   graphiQLEndpoint = '/graphql',
```

Comment:
Note that I may add another option so can have graphql endpoint and playground graphiql at two different urls.